enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/SøknadBarnetilsyn.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.familie.kontrakter.ef.søknad
+
+data class SøknadBarnetilsyn(val personalia: Søknadsfelt<Personalia>,
+                             val innsendingsdetaljer: Søknadsfelt<Innsendingsdetaljer>,
+                             val sivilstandsdetaljer: Søknadsfelt<Sivilstandsdetaljer>,
+                             val medlemskapsdetaljer: Søknadsfelt<Medlemskapsdetaljer>,
+                             val bosituasjon: Søknadsfelt<Bosituasjon>,
+                             val sivilstandsplaner: Søknadsfelt<Sivilstandsplaner>? = null,
+                             val barn: Søknadsfelt<List<Barn>>,
+                             val aktivitet: Søknadsfelt<Aktivitet>,
+                             val stønadsstart: Søknadsfelt<Stønadsstart>,
+                             val tidligereFakturaer: Søknadsfelt<Dokumentasjon>? = null,
+                             val barnepassordningFaktura: Søknadsfelt<Dokumentasjon>? = null,
+                             val avtaleBarnepasser: Søknadsfelt<Dokumentasjon>? = null)
```

Comment:
Trenger to dokumentasjonsbehov til - som er frivillig felt: 
- arbeidstid
- spesielleBehov

Eller noe i den duren @blommish ☝️

Reply:
1153d6187bbdaa83b657b4c17adb9ad766c62dd4